### PR TITLE
[VA-5079] Update media xdm fields

### PR DESCRIPTION
**Pull request by lidiaist**
Issue https://github.com/adobe/xdm/issues/1550

### CODE REVIEW

**Comment by lidiaist** [comment]
@mjasrotia Please review. We made a few updates to our experimental media fieldgroups after starting our backend implementation. Thank you!

File: components/datatypes/qoedatadetails.schema.json [hunk 1]
```diff
@@ -15,66 +15,86 @@
...
-          "type": "number",
-          "description": "The number of streams impacted by buffering."
-        },
-        "xdm:error": {
+        "xdm:hasErrorImpactedStreams": {
           "title": "Error Impacted Streams",
-          "type": "number",
-          "description": "The number of streams in which an error event occurred."
+          "type": "boolean",
+          "description": "The number of streams in which an error event occurred (i.e., trackError was called during the playback session, and a type=error heartbeat call was generated)."
```
Comment (prabhum2) [inline components/datatypes/qoedatadetails.schema.json:76]: If this is boolean , Can you update description reflecting latest changes

Comment (lidiaist) [reply components/datatypes/qoedatadetails.schema.json]: @prabhum2 Done, thank you!

**Review by prabhum2 — APPROVED** [review]

**Comment by prabhum2** [comment]
@lidiaist let me know when this is ready to be reviewed and merged . I see more changes after we approved

**Review by mjasrotia — APPROVED** [review]

**Comment by lidiaist** [comment]
@prabhum2 Done. I made the last updates. If I need any further updates in the future, I will create a new PR, so if you could please merge this PR it would be very helpful. Thank you!

**Comment by lidiaist** [comment]
In the last commit, I added the existing endUserIDs and implementationDetails datatypes to our mediaCollection fieldgroup and the implementationDetails datatype to our mediaReporting fieldgroup because this makes it easier for our clients to know which fields to complete for each group of either collection or reporting.  (Currently I added these 2 in the AEP schema as individual fieldgroups separated from mediaCollection and mediaReporting but using them as datatypes inside our fieldgroups makes our job easier). Thank you!

**Pull request merged**